api/serverless.yml

```diff
@@ -548,3 +548,19 @@ functions:
         handler: src/components/event/routes.dailyEventCheck
         events:
             - schedule: cron(0 8 * * ? *) # Every day at 8 a.m.
+
+    # Additional Information
+    createAdditionalInformation:
+        handler: src/components/additionalInformation/routes.create
+        events:
+            - http:
+                  path: ${self:custom.versions.v1}/publication-versions/{id}/additional-information
```

Comment:
For consistencies sake, could keep the name of the id parameter as either id or publicationVersionId as you have in the delete path

Reply:
I'll rename it to publicationVersionId. To refer to the publication version id as just "id" when the handler is in the directory of a different component than publicationVersion is quite misleading I think. This is also done in a few other places so I'll change those at the same time

Reply:
Now pushed here: https://github.com/JiscSD/octopus/pull/570/commits/40c278d22e6d8562668e2f6f09b0a9c2a7c55317